src/web/server/locales/cy/validation.json

```diff
@@ -20,5 +20,5 @@
   "invalidPhoneNumber": "Nisi lacus sed viverra tellus in",
   "invalidEmail": "Interdum velit laoreet, ekil name@example.com",
   "acceptTermsAndConditions": "Dictum fusce ut placerat orci nulla pellentesqu",
-  "selectTextOrEmailChooseChannelForCode": "Nisi lacus sed viverra tellus in"
+  "selectTextOrEmailSendCode": "Nisi lacus sed viverra tellus in"
```

Comment:
How about `selectTextOrEmailMethod`.

Reply:
Gone with selectTextOrEmailChannel as we use channel for the field name.